services/brig/src/Brig/API/Public.hs

```diff
@@ -598,7 +587,7 @@ getRichInfo self user = do
     (Just t1, Just t2) | t1 == t2 -> pure ()
     _ -> throwStd insufficientTeamPermissions
   -- Query rich info
-  fromMaybe mempty <$> lift (wrapClient $ API.lookupRichInfo user)
+  wrapClientE $ API.lookupRichInfo user
```

Comment:
I don't think so. In wai-routes this is a query string with a default value; in servant this is a query string, now with a default value in the handler instead of the route.